### ISSUE TITLE
How I can get authorization -> id from payment responce?

### ISSUE THREAD

**Issue by archana-iron** [plutov/paypal#69]
How can I get authorization -> id from this type of response
 ```
{
            "intent": "authorize",
            "payer": {
                "payment_method": "credit_card",
                "funding_instruments": [
                    {
                        "credit_card": {
                            "payer_id": "3888888888888888888",
                            "number": "xxxxxxxxxxxx8888",
                            "type": "DISCOVER",
                            "expire_month": "10",
                            "expire_year": "2028",
                            "first_name": "ABC"
                        }
                    }
                ]
            },
            "transactions": [
                {
                    "amount": {
                        "currency": "USD",
                        "total": "4.00",
                        "details": {}
                    },
                    "description": "This is the authorize payment transaction description.",
                    "related_resources": [
                        {
                            "authorization": {
                                "amount": {
                                    "currency": "USD",
                                    "total": "4.00",
                                    "details": {}
                                },
                                "create_time": "2018-10-12T11:23:51Z",
                                "update_time": "2018-10-12T11:23:53Z",
                                "state": "authorized",
                                "parent_payment": "PAY-37J37J37J37J37J37J37J37J",
                                "id": "4FR34FR34FR34FR3A",
                                "valid_until": "2018-11-10T11:23:51Z",
                                "links": [
                                    {
                                        "href": "https://api.sandbox.paypal.com/v1/payments/authorization/4FR34FR34FR34FR3A",
                                        "rel": "self",
                                        "method": "GET"
                                    },
                                    {
                                        "href": "https://api.sandbox.paypal.com/v1/payments/authorization/4FR34FR34FR34FR3A/capture",
                                        "rel": "capture",
                                        "method": "POST"
                                    },
                                    {
                                        "href": "https://api.sandbox.paypal.com/v1/payments/authorization/4FR34FR34FR34FR3A/void",
                                        "rel": "void",
                                        "method": "POST"
                                    },
                                    {
                                        "href": "https://api.sandbox.paypal.com/v1/payments/payment/PAY-37J37J37J37J37J37J37J37J",
                                        "rel": "parent_payment",
                                        "method": "GET"
                                    }
                                ]
                            }
                        }
                    ]
                }
            ],
            "id": "PAY-37J37J37J37J37J37J37J37J",
            "create_time": "2018-10-12T11:23:51Z",
            "state": "approved",
            "update_time": "2018-10-12T11:23:53Z",
            "links": [
                {
                    "href": "https://api.sandbox.paypal.com/v1/payments/payment/PAY-37J37J37J37J37J37J37J37J",
                    "rel": "self",
                    "method": "GET"
                }
            ]
        }
    }
}
```
I need to get this id "4FR34FR34FR34FR3A" to capture payment, how can I can I get this id from this response in golang?

**Comment by plutov** [comment]
Hi @archana-iron I just check and you can get it from Go:
```
	Authorization struct {
		Amount                    *Amount    `json:"amount,omitempty"`
		CreateTime                *time.Time `json:"create_time,omitempty"`
		UpdateTime                *time.Time `json:"update_time,omitempty"`
		State                     string     `json:"state,omitempty"`
		ParentPayment             string     `json:"parent_payment,omitempty"`
		ID                        string     `json:"id,omitempty"`
		ValidUntil                *time.Time `json:"valid_until,omitempty"`
		Links                     []Link     `json:"links,omitempty"`
		ClearingTime              string     `json:"clearing_time,omitempty"`
		ProtectionEligibility     string     `json:"protection_eligibility,omitempty"`
		ProtectionEligibilityType string     `json:"protection_eligibility_type,omitempty"`
	}
```

**Comment by plutov** [comment]
Fixed in v1.1.4
